dl/src/main/scala/com/intel/analytics/bigdl/optim/Optimizer.scala

```diff
@@ -71,6 +73,18 @@ abstract class Optimizer[T: ClassTag, D](
     this
   }
 
+  def setValidation(trigger: Trigger, sampleRDD: RDD[Sample[T]],
+      vMethods : Array[ValidationMethod[T]], batchSize: Int)
+  : this.type = {
```

Comment:
Change to:

val dataSet = DataSet.rdd(sampleRDD) -> SampleToBatch(batchSize)
setValidation(trigger, dataSet, vMethods)